packages/react-core/src/components/TimePicker/__tests__/TimePicker.test.tsx

```diff
@@ -86,6 +86,27 @@ describe('TimePicker', () => {
         expects: { hour: 23, minutes: 59, seconds: null }
       });
     });
+
+    test('should call onChange when pressing Enter', async () => {
```

Comment:
Would we want to skip this test for and update it as a followup?

Reply:
what is the issue so I can note it in my PR.

Reply:
It's just expecting the wrong time on GitHub for some reason. I pulled your branch locally and the test passes fine there, so just another time related issue with tests